doc/api/permissions.md

```diff
@@ -10,6 +10,12 @@ be accessed by other modules.
   This can be used to control what modules can be accessed by third-party
   dependencies, for example.
 
+* [Process-based permissions](#process-based-permissions) control the Node.js
+  process's access to resources (for now, only file system).
```

Comment:
I wouldn't mention it here, since they are just a boolean (either you allow it or not). I plan to add a few settings such as "which process would you allow".

Reply:
I just suggest to remove the "only file system", which to me implies that nothing else is supported.